handler/push.go

```diff
@@ -36,6 +37,9 @@ import (
 	"github.com/prometheus/pushgateway/storage"
 )
 
+// Regex labelRegex is used to check the validity of labelnames in splitLabels.
+var labelRegex = regexp.MustCompile("^[a-zA-Z_][a-zA-Z0-9_]*$")
```

Comment:
You should use `LabelNameRE` from https://github.com/prometheus/common/blob/master/model/labels.go#L84 rather than duplicating this